### DISCUSSION TITLE
Project status, Icon development + letting you know more about me

### DISCUSSION THREAD

**Discussion by GiorgosXou** [GiorgosXou/TUIFIManager#90] [Announcements]
😅 *(In case you thought so...)*  **No, I haven't abandoned the project... I'm** just *(slightly more)* actively **looking for a job** right now, **while** also being focused on **developing [another project](https://github.com/GiorgosXou/NeuralNetworks)**. Yesterday, **I've started [expirimenting with latex](https://github.com/GiorgosXou/our-neovim-setup/commit/a048aa5c1848be5b79cb55092244083503300ba9)**, *(for the main purpose of)* making a more profesional-resume and so I've thought... "Hey! why don't you make an icon for tuifi?" **and here we are 😄:**

|ICON 1 | GIF | ICON 2|
|:---:|:---:|:---:|
|![image](https://github.com/GiorgosXou/TUIFIManager/assets/11987271/34dec529-7f27-46b6-abfc-8453f7d4ad78)|![tuifi latext icon Peek 2024-03-06 15-33](https://github.com/GiorgosXou/TUIFIManager/assets/11987271/46c3be16-e7c1-4d46-8327-7b88f8558046)|![image](https://github.com/GiorgosXou/TUIFIManager/assets/11987271/f97cc675-32a7-4f7f-85a6-0f8c598420a9)|

***Here's my priorities, for when I get back to it:***
- [Sixel support](https://github.com/GiorgosXou/TUIFIManager/issues/86#issuecomment-1723370357)
- [Drag & drop into GUIs](https://github.com/GiorgosXou/TUIFIManager/issues/21)

**Which ICON you prefer the most?** *(1, 2 or something different and why? let me know)*

[![Donate](https://img.shields.io/badge/Donate-PayPal-blue.svg)](https://www.paypal.com/donate/?hosted_button_id=QNQN23M55EJVS)

**Comment by tijko** [comment]
~~Anyway to link the images for a closer look?~~

Right now they're `https://private-user-images.githubusercontent.com` <sup>and all is well...</sup>

Edit: it was an issue with my network, I had thought the links were routing through a firewall.
